### PR TITLE
Migration to buffer with pts dts

### PR DESCRIPTION
**Pull request by andpodob**
Updates plugin to use `:pts` and `:dts` fields in `Membrane.Buffer`

### CODE REVIEW

File: lib/membrane/rtp/serializer.ex [hunk 1]
```diff
@@ -85,10 +85,11 @@ defmodule Membrane.RTP.Serializer do
       extension: Map.get(rtp_metadata, :extension)
     }
 
-    packet = %RTP.Packet{header: header, payload: payload}
+    packet = %RTP.Packet{header: header, payload: buffer.payload}
     payload = RTP.Packet.serialize(packet, align_to: state.alignment)
 
     buffer = %Buffer{
+      pts: Buffer.get_dts_or_pts(buffer),
```
Comment (mat-hek) [inline lib/membrane/rtp/serializer.ex:92]: Here we potentially set dts as pts, and set dts to nil... why?

Comment (andpodob) [reply lib/membrane/rtp/serializer.ex:92]: Yes, you're right, that's probably messed up, I need to rethink that a bit.

**Review by Qizot — APPROVED** [review]

File: lib/membrane/rtp/serializer.ex [hunk 2]
```diff
@@ -77,11 +77,11 @@ defmodule Membrane.RTP.Serializer do
...
   @impl true
-  def handle_process(:input, %Buffer{payload: payload, metadata: metadata}, _ctx, state) do
-    {rtp_metadata, metadata} = Map.pop(metadata, :rtp, %{})
+  def handle_process(:input, buffer, _ctx, state) do
+    {rtp_metadata, metadata} = Map.pop(buffer.metadata, :rtp, %{})
 
     rtp_offset =
-      metadata.timestamp
+      buffer.pts
       |> Ratio.mult(state.clock_rate)
```
Comment (mat-hek) [inline lib/membrane/rtp/serializer.ex:85]: ratio seems not needed

**Review by mat-hek — APPROVED** [review]

**Comment by mat-hek** [comment]
also Ratio not needed in VAD and jitterbuffer/timestamp_calculation_test.exs

**Pull request merged**